sdk/src/main/scala/coop/rchain/sdk/primitive/MapSyntax.scala

```diff
@@ -0,0 +1,25 @@
+package coop.rchain.sdk.primitive
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+
+trait MapSyntax {
+  implicit def mapSyntax[K, V](map: Map[K, V])            = new MapOps[K, V](map)
+  implicit def mutableMapSyntax[K, V](map: TrieMap[K, V]) = new MutableMapOps[K, V](map)
+}
+
+final case class MapOps[K, V](map: Map[K, V]) extends AnyVal {
+  def getUnsafe(k: K): V = {
+    val vOpt = map.get(k)
+    require(vOpt.isDefined, s"No key $k in a map.")
+    vOpt.get
+  }
+}
+
+final case class MutableMapOps[K, V](map: mutable.Map[K, V]) extends AnyVal {
+  def getUnsafe(k: K): V = {
+    val vOpt = map.get(k)
+    require(vOpt.isDefined, s"No key $k in a map.")
+    vOpt.get
+  }
+}
```

Comment:
Please define syntax in a recommended way with _private val_ parameter and _regular_ class.

Here is the example from cats.
https://github.com/typelevel/cats/blob/cf8a87a73215/core/src/main/scala/cats/syntax/monad.scala#L22-L43